src/game.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const readline = require('readline');
+const getRandomNumber = require('./randomizer');
+const checkCattle = require('./checkCattle');
+const myConsole = require('console');
+
+const win = `
+              (      )
+              ~(^^^^)~
+               ) @@ \\~_          |\\
+              /     | \\        \\~ /
+             ( 0  0  ) \\        | |
+              ---___/~  \\       | |
+               /'__/ |   ~-_____/ |
+o          _   ~----~      ___---~
+  O       //     |         |
+         ((~\\  _|         -|
+   o  O //-_ \\/ |        ~  |
+        ^   \\_ /         ~  |
+               |          ~ |
+               |     /     ~ |
+               |     (       |
+                \\     \\      /\\
+               / -_____-\\   \\ ~~-*
+               |  /       \\  \\
+               / /         / /
+             /~  |       /~  |
+             ~~~~        ~~~~
+`;
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+async function game() {
+  myConsole.log('፨ Buls and cows ፨');
+
+  const number = getRandomNumber(1000, 9999).toString();
```

Comment:
Here's you have the next logical bug:
![image](https://user-images.githubusercontent.com/45945732/233980775-53e23a3e-2a11-42d5-ba38-f20000f49a02.png)
All digits in the number must be unique.

Reply:
Also when I type duplicates in the original game I should get an error that my version contains duplicated digits, but it wasn't specified in the task, so it's up to you to do. But you see from the screenshot that in such case it's easy to guess number, because I just try 
1111
2222
3333
...
and I can see positions for numbers because of word bull  :)

Reply:
Also, it's good to check length of what user types